conversions/decimal_to_binary_recursion.py

```diff
@@ -1,25 +1,23 @@
-""" Convert decimal values to binary using recursion method """
-
-
-def bin_recursive(decimal: int) -> str:
+def binary_recursive(decimal: int) -> str:
     """
-    The funtion takes in a positive integer value
+    This takes in a positive integer value
     and returns its binary equivalent.
-    >>> bin_recursive(1000)
+    >>> binary_recursive(1000)
     '1111101000'
-    >>> bin_recursive("72")
+    >>> binary_recursive("72")
     Traceback (most recent call last):
     ...
     TypeError: unsupported operand type(s) for //: 'str' and 'int'
-    >>> bin_recursive("number")
+    >>> binary_recursive("number")
     Traceback (most recent call last):
     ...
     TypeError: unsupported operand type(s) for //: 'str' and 'int'
     """
     # Initialize exit base of the recursion function
     if decimal == 1 or decimal == 0:
         return str(decimal)
-    return bin_recursive(decimal // 2) + str(decimal % 2)
+    result = binary_recursive(decimal // 2) + str(decimal % 2)
+    return str(result)
```

Comment:
I am not a fan of this change.  Creating a variable that has a lifetime of just two lines is not helpful unless it shortens a very long line or the variable name documents something that is not obvious.  `result` in this context does neither.  `binary_recursive()` is documented as returning a str so it is OK to add a str to it.